docs/one-click-integrations/jira-integration.md

```diff
@@ -15,6 +15,9 @@ To set up the Jira integration:
 
     ![Installing the Pulse Jira app](images/jira-installing.png)
 
+    !!! warning
```

Comment:
Presenting this as a warning may be too strong, as this is in fact a limitation for a very specific use case. I'd present this as a note instead:

```suggestion
    !!! note
```